pylint/utils.py

```diff
@@ -693,6 +693,16 @@ def register_messages(self, checker):
             self._messages[msg.symbol] = msg
             self._alternative_names[msg.msgid] = msg
             for old_id, old_symbol in msg.old_names:
+                assert (old_id not in self._alternative_names or
+                        self._alternative_names[old_id] == msg), \
+                        'Old message id %r is already defined' % old_id
+                assert (old_symbol not in self._messages or
+                        self._messages[old_symbol] == msg), \
+                        'Old message symbol %r is already defined ' \
+                        'as a current symbol' % old_symbol
+                assert (old_symbol not in self._alternative_names or
+                        self._alternative_names[old_symbol] == msg), \
+                        'Old message symbol %r is already defined' % old_symbol
```

Comment:
OK, so:

commit 1: Create exceptions.py, add InvalidMessageError and the error checking that will raise this error.
commit 2: Move UnknownMessage and EmptyReport to exceptions.py as well
commit 3: Rename to UnknownMessageError and EmptyReportError throughout the code

Does that sound right?